packages/vue-app/template/components/nuxt-link.client.js

```diff
@@ -0,0 +1,85 @@
+<%= isTest ? '// @vue/component' : '' %>
+import Vue from 'vue'
+
+const requestIdleCallback = window.requestIdleCallback ||
+  function (cb) {
```

Comment:
Actually it throws an error:
![screenshot 2018-12-24 at 15 38 51](https://user-images.githubusercontent.com/904724/50401854-09a5ef00-0792-11e9-9249-e592e70425c6.png)

Also, since we don't need `this` here it would be useless to have an arrow function :)